### ISSUE TITLE
Sometimes subtitles aren't visible while using pause and `sub-seek -1`

### ISSUE THREAD

**Issue by kelciour** [mpv-player/mpv#5365]
Sometimes `sub-seek -1` with external subtitles seek at the position of the video that 1 frame earlier and subtitles aren't visible until the next frame. At the same time, `sub-seek 1` works correctly and subtitles are always visible.

### mpv version and platform

Windows 7 and mpv 0.28.0 (C) 2000-2017 built on Mon Dec 25 01:08:49 CET 2017

### Reproduction steps

Open some video with external .srt subtitles, set the video on pause and use Ctrl+Left to seek to the previous subtitle.

### Expected behavior

Subtitles are always visible.

### Actual behavior

Sometimes subtitles aren't visible. If press . to step one frame forward the subtitles will be shown.

In both cases (when subtitles are visible and when subtitles aren't visible while using `sub-seek -1`) the timestamps for the same subtitle in `queuing seek to` in the log file are slightly different when performing `sub-seek -1` and `sub-seek 1`.

### Video

https://0x0.st/sZfl.mp4

**Comment by wm4** [comment]
That's because it seeks slightly before the start of the subtitle. It's supposed to fix the thing that doing backwards sub-seeks twice sometimes gets "stuck".

**Comment by kelciour** [comment]
Oh, thanks, I thought that it was some minor bug that can be easily fixed. Nevertheless, I've added some basic version of `sub-seek -1` in [sub-replay.lua](https://github.com/kelciour/mpv-scripts/blob/master/sub-replay.lua), but the strange thing is that I used `seek` command with `absolute+exact` parameters but still got the same results as with `sub-seek -1` and had to add 25 ms to be able to see these subtitles. In that case a seek with the correct subtitles would be one frame ahead, but it's completely fine for me. However, it's not a big deal and I think this issue can be closed.
